### DISCUSSION TITLE
How to get an overview of all my changeset comments?

### DISCUSSION THREAD

**Discussion by ERYpTION** [openstreetmap/iD#10505] [Q&A]
I’m trying to find a way to get an overview of all the comments I’ve made on other people's changesets. While I can see my own changesets on my OSM profile, I’m struggling to locate my comments on changesets created by others.

Is there a tool or method that allows me to see all my comments on other users' changesets in one place?

**Comment by Dimitar5555** [comment]
Check out https://hdyc.neis-one.org. You have to enter your username in the box (and possibly login with OSM before that). There you will see "Changeset discussions: Participated in X ....". Clicking on the number will list all of your changeset comments.

P.S. This repository is for the web OSM editor. It doesn't deal with changeset comments. A proper place to post such question would've been the [community forum](https://community.openstreetmap.org).
